tests/vmi_multus_test.go

```diff
@@ -512,6 +386,181 @@ var _ = Describe("Multus Networking", func() {
 	})
 })
 
+var _ = Describe("SRIOV", func() {
+
+	flag.Parse()
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	var vmiOne *v1.VirtualMachineInstance
+
+	tests.BeforeAll(func() {
+		tests.SkipIfNoMultusProvider(virtClient)
+		tests.BeforeTestCleanup()
```

Comment:
maybe you can add use the `tests.SkipIfNoSriovDevicePlugin(virtClient)` and change it to `FailIfNoSriovDevicePlugin(virtClient)`

Reply:
@SchSeba if I make it fail, I think it will break execution for when the plugin is not installed (which is a valid configuration assuming you don't have SR-IOV hardware in the cluster). Or is there a way to make a suite disabled by default, in which case we would "focus" it only when we need to execute it (as part of SR-IOV enabled job)?